goverance.md

```diff
@@ -0,0 +1,67 @@
+- Members going on leave for 1-3 months *MAY* work with other members to identify a temporary replacement.
+- Members of a role *SHOULD* remove any other members that have not communicated a leave of absence and either cannot be reached for more than 1 month or are not fulfilling their documented responsibilities for more than 1 month. This may be done through a [super-majority] vote of members, or if there are not enough *active* members to get a super-majority of votes cast, then removal may occur through a [super-majority] vote between Chairs and Technical Leads.
+- Membership disagreements may be escalated to the WG Chairs.  WG Chair membership disagreements may be escalated to the CNCF Steering Committee.
+- Members *MAY* decide to step down at anytime and propose a replacement.  
+- The group will use lazy consensus amongst other members with fallback on majority vote to accept proposal.  The candidate *SHOULD* be supported by a majority of SIG Members or Subproject Contributors (as applicable).
+- Members *MAY* select additional members through a [super-majority] vote amongst members. This *SHOULD* be supported by a majority of WG Members.
+
+### Role of Chairs
+
+* Primary role of chairs is to run operations and processes governing the WG.
+* The Chairs are responsible for providing technical guidance, for bringing up proposals that have been submitted, and for asking for new proposals to be made.
+* A chair will serve in their role for 2 years from when they started in the role, and *MAY* run again indefinitely.
+* To fill a vacancy for a new chair, the group will announce with at least two week’s notice that a vote for chair will occur. Each company will have one vote. Any group member is eligible to run for Chair.
+* There will be three chairs and their membership will be tracked in [`wg.yaml`]().
+
+
+### Role of Technical Leads
+
+Technical Leads will lead larger streams of work that require sustained technical leadership. Any new proposal should have a TL working as an active sponsor of the proposal, guiding the proposal with the engagement of community. The general list of activities for TL are
+  - Establish new subprojects
+  - Decommission existing subprojects
+  - Resolve cross-subproject technical issues and decisions
+
+There will be two to three TLs and their membership will be tracked in [`wg.yaml`](). TLs are elected by one of the chairs nominating a candidate and creating a PR that adds the TL to [`wg.yaml`](). Members are invited to comment and vote according to the regular proposal process described below.
+
+## Creating, discussing and accepting proposals
+
+Each proposal is unique and might deviate slightly from the process below. For example, a smaller project may each step. For example, a small addition may not require establishing completion criteria. In general, we encourage this process below to be followed to ensure that contributions are in line with the goals of the Working Group.
```

Comment:
The two sentences starting with "For example" could be clarified - I'm not sure what they're trying to say.

Reply:
Great catch! Fixed this